batchflow/models/tf/layers/conv_block.py

```diff
@@ -1,44 +1,31 @@
 """ Contains convolution layers """
+import logging
 import numpy as np
 import tensorflow as tf
+import tensorflow.keras.layers as K #pylint: disable=import-error
```

Comment:
There is no common style for this alias but I would rather use something in lower case and not a single letter, `kl`, for example

Reply:
Probably the best way is to directly import needed layers like `Conv2D` from `tensorflow.keras.layers`